src/main/java/com/hades/login/LoginController.java

```diff
@@ -26,15 +26,18 @@
 
 	@PermitEndpoint
 	@RequestMapping(path = "/login", method = POST)
-	public LoginInfoDTO login(@RequestBody @Valid LoginInfo loginInfoToAuthenticate) {
```

Comment:
parace que precisa mesmo do `@RequestBody`, sem ele estou levando um `400` no angular com a mensagem `Validation failed for object='loginInfo'. Error count: 2`